src/Login/components/Login.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Image, View, Text, TouchableOpacity, TextInput } from 'react-native';
+import { PropTypes } from 'prop-types';
+
+import styles from '../components/Login.style';
```

Comment:
This can be shortened to `Login.style`

Reply:
Oh okay.

I think in the future we can just name them all `style.js`, since they're already in the same folder, there's really no ambiguity.

Sorry about that haha